### PR TITLE
New package: ScaniaLabs.Miroirs version 1.0.020216.08

### PR DESCRIPTION
**Pull request by denismattos**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/187641)

### CODE REVIEW

File: manifests/s/ScaniaLabs/Miroirs/1.0.020216.08/ScaniaLabs.Miroirs.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- /help
+ProductCode: '{C2BBC4E5-A363-43F6-B008-278BF00D6559}'
+AppsAndFeaturesEntries:
+- DisplayName: Miroirs
+  Publisher: Scania Labs
+  DisplayVersion: 1.0.7
+  ProductCode: '{C2BBC4E5-A363-43F6-B008-278BF00D6559}'
+  UpgradeCode: '{352AF39F-279E-4B70-8958-BA9A20FF6142}'
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/s/ScaniaLabs/Miroirs/1.0.020216.08/ScaniaLabs.Miroirs.installer.yaml:24]:
```suggestion
- Architecture: x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Automatic Validation ended with:
> Manifest Error: Duplicate installer entry found.
WinGetSvc_Validation • Pull Request Validation • Validate Manifest

Purely guessing that one of these installers should be a different architecture.

**Comment by denismattos** [comment]
I'm sorry for not addressing this earlier, but I couldn't log into GitHub lately.  
I am at a loss about the raised issue.  
Automatic Validation reported a duplicate installer, but I'm quite sure I only provided Komac with 2 URLs for the same file.  
The hashes matched.

Unless this is due to [my own, unsuccessful attempt to create this same manifest, back in June](https://github.com/microsoft/winget-pkgs/pull/157982#event-13173307210).  
If that's the case, IDK how to solve the issue.  
I used Komac this time around exactly to automate the process and avoid running into the same problems as last time and IDK how I can withdraw/delete my previous attempt.

I believe I may need help with this from one of the maintainers. 🫤

**Comment by stephengillie** [comment]
Hi @denismattos,

No worries on the delay. Your previous PR is closed and has no bearing on this PR. The issue is entirely within the manifest for this PR.  

As I suggested above, I believe the issue is that one of the installers is supposed to have `x86` architecture. I believe the way to solve the issue is to make the suggested change.

**Comment by stephengillie** [comment]
> Manifest Error: Duplicate installer entry found.

**Comment by denismattos** [comment]
Hey, @stephengillie.  
Thx for the attention. 🙂
> No worries on the delay. Your previous PR is closed and has no bearing on this PR. The issue is entirely within the manifest for this PR.

I guess it took me too much time, after that, either way, uh? 😅  
Sorry again. Life's been hectic and I can't access GH nearly as much as I'd like.
> As I suggested above, I believe the issue is that one of the installers is supposed to have `x86` architecture. I believe the way to solve the issue is to make the suggested change.

AFAI'm aware, both .msi I supplied are identical. I even checked the hashes on the PR and they match.  
What I believe happened is I fumbled something when answering Komac's queries and fooled it into believing there were 2 separate installers when there's actually only the one and a mirror.  
I may be able to look into this later this week and push again. If I'm right, it should be really simple.

Thx again for the attention and the feedback.  
Cheers!

**Pull request closed without merging**